### PR TITLE
Fix mashing script after get_from_git refactor

### PR DESCRIPTION
**Pull request by ekohl**
This parameter was not used and callers weren't passing it in.

Fixes: 3f4ee09d908dc7f947f451cad8c3ba68d18157d6

### CODE REVIEW

File: koji/collection-mash-split.py [hunk 1]
```diff
@@ -125,12 +123,13 @@ def createrepo(self, path, comps=None, checksum=None):
...
         @param path: path to repository
         @type path: str
-        @param comps: comps file (not mandatory)
+        @param comps: comps file content (not mandatory)
         @type comps: str
         """
         cmd = "createrepo --pretty --quiet --database "
         if comps:
-            cmd += "--groupfile %s " % comps
+            with tempfile.NamedTemporaryFile() as comps_file:
```
Comment (evgeni) [inline koji/collection-mash-split.py:131]:
```suggestion
            with tempfile.NamedTemporaryFile() as comps_file:
                comps_file.write(comps)
```

Comment (evgeni) [reply koji/collection-mash-split.py:131]: and probably .close() so that it's fully written?

Comment (ekohl) [reply koji/collection-mash-split.py]: I just called `flush()` instead.

**Review by evgeni — APPROVED** [review]

**Comment by ekohl** [comment]
Deployed and verified I could mash foreman nightly.

**Pull request merged**